### DISCUSSION TITLE
Does netflix-community version jars get uploaded to MVN central repo?

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor-community#132] [Q&A]
We have been downloading Conductor server jar from MVN repo to run in our organization. We never built locally. 

As postgres and mysql support is taken off from main build , is community version uploaded to mvn central repo?

**Comment by manan164** [comment]
Hi @shekarpcs , Yes they are uploaded. Please see the [guide](https://github.com/Netflix/conductor-community) in order to understand how to pull it.
